updater/aws_test.go

```diff
@@ -1027,7 +1027,7 @@ func TestUpdateInstanceErr(t *testing.T) {
 			containerInstanceID: "cont-inst-id",
 		})
 		require.Error(t, err)
-		assert.ErrorIs(t, err, waitExecErr)
+		assert.Contains(t, err.Error(), "failed to wait ssm execution complete")
```

Comment:
Does it gives error without this change? I am trying to understand how the code change to affect this assertion.

Reply:
Before I changed the error handling to pass `err` to the channel I had it wrapping the error which did cause the test to fail, so I changed it to `Contains` like other cases which validate wrapped errors. This escaped me when I made the last revision. I'll switch it back to `ErrorIs` since thats working now.

Reply:
Similarly, there is one more similar assertion which might wanna fix.

Reply:
For sure, already planning on changing it too :)